README.md

```diff
@@ -4,7 +4,7 @@
 
 `Injectable` is an opinionated and declarative [Dependency Injection](https://en.wikipedia.org/wiki/Dependency_injection) library for ruby.
 
-It is being used in production (under ruby 2.5.1) in [RubiconMD](https://github.com/rubiconmd) and was extracted from its codebase.
+It is being used in production (under ruby 2.7.6) in [RubiconMD](https://github.com/rubiconmd) and was extracted from its codebase.
```

Comment:
You can put we're using ruby 3.1 haha

Reply:
👌  fixed with https://github.com/rubiconmd/injectable/pull/29/commits/06540949cc05dab345999da7d84d8f444143adfe